From d399bfbb3b0c6e1c06e2d78f494a5784db271e8e Mon Sep 17 00:00:00 2001
From: Pavan S <psreenivas@habana.ai>
Date: Mon, 28 Oct 2024 15:48:41 +0200
Subject: [PATCH] [SW-204744] gaudi3: remove redefined macro

This patch removes CPLD_ADDR macro as this macro
is added in https://gerrit.habana-labs.com/#/c/459049/
which fixed refined warning during compilation.

Signed-off-by: Pavan S <psreenivas@habana.ai>
Change-Id: I2f301d37d7d824eec060323f6aa999defaaf456f
---

diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index 55dcb3b..bb1a6fe 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -842,7 +842,6 @@
 #define I2C_RETRY_COUNT				20
 
 #if DT_NODE_HAS_STATUS(CPLD_DEV, okay)
-#define CPLD_ADDR	DT_REG_ADDR(CPLD_DEV)
 static void preboot_check_enter_rma_mode(void)
 {
 	int rev;
